docs/flows-format.adoc

```diff
@@ -3,489 +3,277 @@
-
-In conversation tracking, A to B packets dropped counter per conversation
-
-'''
-
-PktDropPackets_BA::
-
-• `Optional` *PktDropPackets_BA*: `number`
-
-In conversation tracking, B to A packets dropped counter per conversation
-
-'''
-
-PktDropBytes::
-
-• `Optional` *PktDropBytes*: `number`
-
-Number of bytes dropped by the kernel
-
-'''
-
-PktDropBytes_AB::
-
-• `Optional` *PktDropBytes_AB*: `number`
-
-In conversation tracking, A to B bytes dropped counter per conversation
-
-'''
-
-PktDropBytes_BA::
-
-• `Optional` *PktDropBytes_BA*: `number`
-
-In conversation tracking, B to A bytes dropped counter per conversation
-
-'''
-
-DnsId::
-
-• `Optional` *DnsId*: `number`
-
-DNS record id
-
-'''
-
-DnsFlags::
-
-• `Optional` *DnsFlags*: `number`
-
-DNS flags for DNS record
-
-'''
-
-DnsFlagsResponseCode::
-
-• `Optional` *DnsFlagsResponseCode*: `string`
-
-Parsed DNS header RCODEs name
-
-'''
-
-DnsLatencyMs::
-
-• `Optional` *DnsLatencyMs*: `number`
-
-Calculated time between response and request, in milliseconds
-
-'''
-
-TimeFlowStartMs::
-
-• *TimeFlowStartMs*: `number`
-
-Start timestamp of this flow, in milliseconds
-
-'''
-
-TimeFlowEndMs::
-
-• *TimeFlowEndMs*: `number`
-
-End timestamp of this flow, in milliseconds
-
-'''
-
-TimeReceived::
-
-• *TimeReceived*: `number`
-
-Timestamp when this flow was received and processed by the flow collector, in seconds
-
-'''
-
-TimeFlowRttNs::
-
-• `Optional` *TimeFlowRttNs*: `number`
-
-Flow Round Trip Time (RTT) in nanoseconds
-
-'''
-
-_HashId::
-
-• `Optional` *_HashId*: `string`
-
-In conversation tracking, the conversation identifier
-
-'''
-
-_IsFirst::
-
-• `Optional` *_IsFirst*: `string`
-
-In conversation tracking, a flag identifying the first flow
-
-'''
-
-numFlowLogs::
-
-• `Optional` *numFlowLogs*: `number`
-
-In conversation tracking, a counter of flow logs per conversation
-
-
-== Enumeration: FlowDirection
-
-
-Ingress::
-
-• *Ingress* = `"0"`
-
-Incoming traffic, from the node observation point
-
-'''
-
-Egress::
-
-• *Egress* = `"1"`
-
-Outgoing traffic, from the node observation point
-
-'''
-
-Inner::
-
-• *Inner* = `"2"`
-
-Inner traffic, with the same source and destination node
-
-
-== Enumeration: InterfaceDirection
-
-
-Ingress::
-
-• *Ingress* = `"0"`
-
-Incoming traffic, from the network interface observation point
-
-'''
-
-Egress::
-
-• *Egress* = `"1"`
-
-Outgoing traffic, from the network interface observation point
+This is the specification of the network flows format. That format is used when a Kafka exporter is configured, for Prometheus metrics labels as well as internally for the Loki store.
+
+The "Filter ID" column shows which related name to use when defining Quick Filters (see `spec.consolePlugin.quickFilters` in the `FlowCollector` specification).
+
+The "Loki label" column is useful when querying Loki directly: label fields need to be selected using link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
```

Comment:
Do we still need to write this too:
> If you are reading this specification as a reference for the Kafka export feature, you must treat all _Labels_ and _Fields_ as regular fields and ignore any distinctions between them that are specific to Loki.

Reply:
No I removed that because now it's not organized by Fields and Labels anymore - it's directly written for Kafka exporter users

Reply:
Thanks Joel. I re-copied your latest IBMSG tweaks to my PR.

Reply:
re-copied and pushed latest to my PR.